ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -8,17 +8,17 @@ ext {
     // String property to store the proper name of the mediation network adapter.
     adapterName = "Vungle"
     // String property to store version name.
-    stringVersion = "6.10.2.0"
+    stringVersion = "6.10.3.0"
     // String property to store group id.
     stringGroupId = "com.google.ads.mediation"
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
as the adapter is still at API 30 and we cant bump that until Google wants to, does it even help bumping this on Test App side ?